examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -70,3 +71,18 @@
 
 ax.set_title('Four left-temporal magnetometers')
 plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
hmm ... could it be that the proj was computed on the unfiltered data -- so when you apply it on the filtered data, it creates this issue. It is because the subspace of the filtered data is different from the subspace of unfiltered data. I think we should fix this -- although I'm not sure what's the right fix. Should you also filter the proj? Any ideas @Eric89GXL or @agramfort  ?